README.md

```diff
@@ -13,34 +13,32 @@ A safe, extensible ORM and Query Builder for Rust
 [Homepage](http://diesel.rs)
 
 Diesel gets rid of the boilerplate for database interaction and eliminates
-runtime errors, without sacrificing performance. It takes full advantage of
+runtime errors without sacrificing performance. It takes full advantage of
 Rust's type system to create a low overhead query builder that "feels like
-Rust".
+Rust."
 
 ## Getting Started
 
-You can find an extensive Getting Started tutorial at
+Find our extensive Getting Started tutorial at
 [http://diesel.rs/guides/getting-started](http://diesel.rs/guides/getting-started).
-Guides on more specific features will be coming soon.
+Guides on more specific features are coming soon.
 
 ## Code of conduct
 
-Anyone who interacts with Diesel in any space including but not limited to
-this GitHub repository is expected to follow our [code of conduct](https://github.com/diesel-rs/diesel/blob/master/code_of_conduct.md)
+Anyone who interacts with Diesel in any space, including but not limited to
+this GitHub repository, must follow our [code of conduct](https://github.com/diesel-rs/diesel/blob/master/code_of_conduct.md)
 
 ## License
 
-Licensed under either of
+Licensed under either of these:
```

Comment:
Interesting change, I like it. I'm fairly certain the original text of this section (incl. Contributing) was copypasta from a template, though. (There was an initiative to relicense a bunch of code under MIT/Apache2)